packages/ResponsesAccordion/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@paprika/responses-accordion",
+  "version": "0.1.0",
+  "description": "Responses Accordion component",
+  "author": "@paprika",
+  "main": "lib/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/ResponsesAccordion"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/collapsible": "^0.1.1",
+    "@paprika/icon": "^0.1.3",
+    "@paprika/l10n": "^0.1.2",
```

Comment:
I actually depend on a future version of the L10n for the key I added. It's a patch update, so it shouldn't really matter, but will lerna also update this dependency here?